# Hypothesis Testing

### H0: Marriage age of male in India is 30
### Ha: Marriage age of male in India is not 30
### 30 is hypothesis mean

In [6]:
mar_age_india = [29,30,25,24,32,27,29,34,33,32,29,28,30,31,32,29]

In [7]:
from scipy import stats

In [8]:
print("Skew: ", stats.skew(mar_age_india))
print("Kurtosis: ", stats.kurtosis(mar_age_india))

Skew:  -0.4457274903384464
Kurtosis:  -0.39217485810892416


## One Sample T-Test

In [9]:
stats.ttest_1samp(mar_age_india, 30)

Ttest_1sampResult(statistic=-0.5447047794019222, pvalue=0.5939629429499389)

pvalue=0.59, pvalue>0.5, so we fail to reject null hypothesis